src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphIncorrect.java

```diff
@@ -41,19 +41,22 @@ class InputJavadocParagraphIncorrect {
     boolean emulated() {return false;}
 
     // 2 violations 4 lines below:
-    //  'tag should be preceded with an empty line.'
     //  'Redundant <p> tag.'
+    //  'tag should be preceded with an empty line.'
     // violation 2 lines below 'tag should be preceded with an empty line.'
     /**<p>Some summary.<p>
      * <p>
      * <p><p>
      * <p>   Some paragraph.<p>*/
-    // violation 2 lines above 'tag should be preceded with an empty line.'
     // 2 violations 2 lines above:
+    //  'tag should be preceded with an empty line.'
+    //  'tag should be preceded with an empty line.'
```

Comment:
columns is correct!!!!
it was my worry that we are not good in columns inside javadoc
![image](https://github.com/user-attachments/assets/31d3d63f-5925-42c3-b67b-84fd7b6c0196)